platformio.ini

```diff
@@ -8,6 +8,9 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
+[platformio]

```

Comment:
TODO: This still needs to include the appropriate HAL source files based on buildtype.